SharedProjects/Babylon2GLTF/GLTFExporter.Animation.cs

```diff
@@ -465,7 +365,7 @@ private GLTFAccessor _createAndPopulateInput(GLTF gltf, BabylonAnimation babylon
                 numKeys++;
                 float inputValue = babylonAnimationKey.frame;
                 if (offsetToStartAtFrameZero) inputValue -= startFrame;
-                inputValue /= Loader.Global.FrameRate;
+                inputValue /= babylonAnimation.framePerSecond;
```

Comment:
this may cause a regression, but it may also be a bug fix, @PatrickRyanMS it may be good to have a test where we have an animation authored with a different frame rate than scene, (if that's possible in both programs)